tools/scalar-generator/project.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "scalar-generator",
+    "$schema": "../../node_modules/nx/schemas/project-schema.json",
+    "sourceRoot": "tools/scalar-generator/src",
+    "projectType": "library",
+    "tags": [],
+    "targets": {
+        "build": {
+            "executor": "@nx/js:tsc",
+            "outputs": ["{options.outputPath}"],
+            "options": {
+                "outputPath": "dist/tools/scalar-generator",
+                "main": "tools/scalar-generator/src/index.ts",
+                "tsConfig": "tools/scalar-generator/tsconfig.lib.json",
+                "assets": [
+                    "tools/scalar-generator/*.md",
+                    {
+                        "input": "./tools/scalar-generator/src",
+                        "glob": "**/!(*.ts)",
+                        "output": "./src"
+                    },
+                    {
+                        "input": "./tools/scalar-generator/src",
+                        "glob": "**/*.d.ts",
+                        "output": "./src"
+                    },
+                    {
+                        "input": "./tools/scalar-generator",
+                        "glob": "generators.json",
+                        "output": "."
+                    },
+                    {
+                        "input": "./tools/scalar-generator",
+                        "glob": "executors.json",
+                        "output": "."
+                    }
+                ]
+            }
+        },
+        "lint": {
+            "executor": "@nx/eslint:lint"
+        },
+        "test": {
+            "executor": "@nx/jest:jest",
+            "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
+            "options": {
+                "jestConfig": "tools/scalar-generator/jest.config.ts"
```

Comment:
I don't see this `jest.config.ts` file, was it not pushed?
Any reason for jest or can it use vitest instead?

Reply:
this was added automatically by nx, I've removed this, we are not testing the generators for now